src/test/scala/rise/core/traverse.scala

```diff
@@ -49,34 +53,65 @@ class traverse extends test_util.Tests {
         { case _: App                                    => () },
         { case primitives.map()                          => () },
         { case _: FunType[_, _]                          => () },
+        { case _: FunType[_, _]                          => () },
```

Comment:
why is the trace different? was the previous implementation bugged or have the trace semantics changed?

Reply:
The trace semantics have changed, we visit far more nodes in the types (e.g. in a function type `A => B` we visit types `A` and `B` too)